readme badges

Carreau opened the pull request:
![capture d ecran 2015-01-08 a 18 47 32](https://cloud.githubusercontent.com/assets/335567/5667335/4b056006-9767-11e4-80cb-9117b3d192e2.png)

Just mostly because otherwise it's hard to find the coverage link. 
So I also added a few other.

We can also be fancy with 

![](https://img.shields.io/badge/Python-2.7%2C%203.3%2B-brightgreen.svg)
![](https://img.shields.io/badge/PHP-Nop, not possible-red.svg)
![](https://img.shields.io/badge/various--style-are--possible-blue.svg?style=flat-square)
![](https://img.shields.io/badge/next release-winterish %283.0%29-orange.svg?style=flat)

Carreau committed:
readme badges

minrk commented on README.rst:
```diff
@@ -1,3 +1,16 @@
+    :target: https://pypi.python.org/pypi/ipython
+
+.. image:: https://img.shields.io/pypi/v/IPython.svg            
+    :target: https://pypi.python.org/pypi/ipython
```
I see there are two copies of pypi

minrk replied:
Ignore me, I see v vs dm

Carreau replied:
For future reader, `v` is version and `dm` download per month.

damianavila commented:
Sounds like a good idea...

ellisonbg commented:
Travis is failing on this one...

takluyver commented:
It seems unlikely that the test failure could be related to this change. I've prodded the relevant job to restart.

minrk committed:
expedite travis

- use sudo: false
- install pandoc, mathjax from rackspace cloudfiles
- install casperjs with npm

takluyver commented:
Still failing. Looks like it's related to #7132, which was recently merged. I think this should be fine to merge, but can you (@Carreau) take a quick look?

minrk committed:
fix highlight test for failed highlighting

not sure how this ever succeeded, but it's failing now

Carreau commented:
Rebased, also included #7407 from @minrk that needed a rebase too.

ellisonbg commented:
Is this ready then?

rgbkrk commented:
Looking good to me.

rgbkrk commented:
The Travis updates I already merged with #7407, so this is just the README updates really now.
